db/migrate/20210602041505_add_column_name_to_items_table.rb

```diff
@@ -0,0 +1,7 @@
+class AddColumnNameToItemsTable < ActiveRecord::Migration[5.2]
+  def change
+    change_table :items do |t|
+      t.string :name
+    end
+  end
+end
```

Comment:
Please rails db:migrate when you pull from main, I forgot the name column for items